src/ContainerOptionFactory.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\Cli;
+
+use Symfony\Component\Console\Input\InputOption;
+
+/**
+ * @internal
+ */
+final class ContainerOptionFactory
```

Comment:
Could revert this one. I've added this in an earlier approach to fix #74 and changed that later on.